### PR TITLE
update travis config to latest rubies

### PR DESCRIPTION
**Pull request by lostapathy**
Updating travis config to cover latest ruby versions

### CODE REVIEW

**Comment by lostapathy** [comment]
Also skip testing rails-4.x with ruby-head.  We're not testing them with ruby-2.4 and they fail anyway.

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ language: ruby
 cache: bundler
 rvm:
   - 2.1
-  - 2.2.4
-  - 2.3.1
-  - 2.4.1
+  - 2.2.8
+  - 2.3.5
+  - 2.4.2
```
Comment (tbrisker) [inline .travis.yml]: maybe just drop the z part so we're always testing with the latest version of each line?

Comment (lostapathy) [reply .travis.yml]: I'm ok with that but went this way to leave the project's existing style alone.  Some projects actively don't like to do it that way.  Occasionally it seems that Travis gets the logic behind that wrong - i.e., for a while this summer 2.2 was getting you 2.2.1 not 2.2.8 like you'd expect.

Comment (tbrisker) [reply .travis.yml]: @danielmorrison any preference?

**Comment by tbrisker** [comment]
@lostapathy Thanks, I pushed a rebase to your branch to fix the merge conflicts, will merge once travis is 🍏

**Pull request merged**